### ISSUE TITLE
ImageDataGenerator does not standartize validation_data

### ISSUE THREAD

**Issue by eyaler** [fchollet/keras#1950]
ImageDataGenerator should apply standartize() on validation_data parameter or at least the docs should mention the the validation data is assumed to already be standartized

**Comment by fchollet** [comment]
There seems to be some confusion between ImageDataGenerator and `fit_generator` here. What do you mean?

**Comment by eyaler** [comment]
e.g. in the cifar10_cnn.py example, if you add featurewise normalization, in ImageDataGenerator, the fit_generator does not apply this normalization to validation_data=X_test

**Comment by asmith26** [comment]
Regarding the doc, I came across this also - it looks like the [ImageDataGenerator doc](http://keras.io/preprocessing/image/) is not being automatically generated, and hence the [actual signature](https://github.com/fchollet/keras/blob/master/keras/preprocessing/image.py#L142) is not being pulled and updating the doc.

I was about to add the ImageDataGenerator class to be automatically updated, however it looks like that class may be soon re-written (see #2446), so I'll hold off for now.

**Comment by eyaler** [comment]
please reopen. this is still an issue. i missed the stale tag.
